client/src/styles/GlobalStyles.tsx

```diff
@@ -135,11 +135,38 @@ const globalStyles: SerializedStyles = css`
   }
 `
 
+const darkMode: SerializedStyles = css`
+  body {
+    filter: invert(100%) hue-rotate(180deg);
+  }
+
+  .prevent-hue-rotate {
+    filter: invert(100%) hue-rotate(180deg);
+  }
+`
+
 const GlobalStyles: React.FC = () => {
+  const [isDarkMode, setIsDarkMode] = React.useState(false)
   return (
     <React.Fragment>
+      <div
+        css={css`
+          position: absolute;
+          right: 25px;
+          top: 25px;
+          font-size: 25px;
+          cursor: pointer;
+          outline: none;
+          user-select: none;
+        `}
+      >
+        <span onClick={() => setIsDarkMode(!isDarkMode)} role="button" className="prevent-hue-rotate">
+          {isDarkMode ? '☀️' : '🌙'}
```

Comment:
Maybe it should be the other way around, as the icon cannot be seen on dark mode.

![Screenshot from 2020-06-09 09-17-31](https://user-images.githubusercontent.com/8309423/84117856-75d1ec80-aa32-11ea-9788-ddbd8ea8888f.png)
![Screenshot from 2020-06-09 09-17-46](https://user-images.githubusercontent.com/8309423/84117860-766a8300-aa32-11ea-9ceb-a16f6b80b951.png)

Reply:
I added some svg images to prevent this